src/test/java/sanghwa/CalculatorTest.java

```diff
@@ -0,0 +1,60 @@
+package sanghwa;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+//한글 메서드명 대신 @DisplayName으로 쓰자
+@DisplayName("calculator tester")
+class CalculatorTest {
+    private final Calculator calculator = new Calculator();
+
+    //@Nested 중첩 클래스 선언, 여러가지 메서드를 사용할 수 있음.
+    @Nested
+    @DisplayName("덧셈")
+    class Add {
+        @Test
+        @DisplayName("일반 덧셈 테스트")
+        void add() {
+            assertEquals(5, calculator.add(2, 3));
+        }
+    }
+
+    @Nested
+    @DisplayName("뺄셈")
+    class Subtract {
+        @Test
+        @DisplayName("일반 뺄셈 테스트")
+        void subtract() {
+            assertEquals(-3, calculator.subtract(3, 6));
+        }
+    }
```

Comment:
아래 메서드들과 동일합니다
`displayName`은 상화님이 생각하기에 어떻게 사용되기를 의도한 어노테이션인 것 같나요?
클래스 이름만으로도 뺄샘의 뜻을 명확히 알 수 있지 않나요?

아래와 같이 작성되어도 해당 메서드가 테스트인가에 대한 의도 전달, 
어떤 동작을 테스트하는지에 대한 의도가 전달되는지 편하게 알 수 있지 않나요?
고민해보고 상화님의 생각을 알려주세요!
```suggestion
    @Nested
    class Subtract {
        @Test
        void subtract() {
            assertEquals(-3, calculator.subtract(3, 6));
        }
    }
```

Reply:
우진님의 코멘트의 전적으로 동의합니다. 
위에서 코멘트했듯이, 리팩토링 단계에서 삭제할 예정입니다.
 주석과 @DisplayName을 _코드의 가독성을 해치지 않고 불필요한 주석과 어노태이션을 작성하지 않도록_ 유의해야겠습니다.